README.md

```diff
@@ -82,19 +82,19 @@ interpolation/decomposition.
 |------------------------|:--------------:|:-------------------:|:------------------------:|
 |Size (gzipped)          | 12.5kb         | 14kb                | 10.5kb                   |
 |Element.animate         | ✔             | ✔                  | ✔                       |
-|Timing input (easings, duration, fillMode, etc.) for animations| ✔ | ✔ | ✔             | 
+|Timing input (easings, duration, fillMode, etc.) for animation effects| ✔ | ✔ | ✔             | 
 |Playback control        | ✔             | ✔                  | ✔                       |
 |Support for animating lengths, transforms and opacity| ✔ | ✔ | ✔                       |
-|Support for Animating other CSS properties| ✔ | ✔            | 🚫                       |
+|Support for animating other CSS properties| ✔ | ✔            | 🚫                       |
 |Matrix fallback for transform animations | ✔ | ✔             | 🚫                       |
-|Animation constructor   | 🚫             | ✔                  | ✔                       |
-|Simple Groups           | 🚫             | ✔                  | ✔                       |
+|KeyframeEffect constructor   | 🚫             | ✔                  | ✔                       |
+|Simple GroupEffects & SequenceEffects           | 🚫             | ✔                  | ✔                       |
 |Custom Effects          | 🚫             | ✔                  | ✔                       |
-|Timing input (easings, duration, fillMode, etc.) for groups</div>| 🚫 | 🚫\* | 🚫         |
+|Timing input (easings, duration, fillMode, etc.) for groups/sequences</div>| 🚫 | 🚫\* | 🚫         |
```

Comment:
> sequences are groups

OK, thanks. I thought we had ditched that terminology. I will change these back and then merge.